src/integration-tests/java/com/webdev/cosmo/cosmobackend/features/UserIT.java

```diff
@@ -0,0 +1,85 @@
+package com.webdev.cosmo.cosmobackend.features;
+
+import com.webdev.cosmo.cosmobackend.BaseTestConfiguration;
+import com.webdev.cosmo.cosmobackend.service.api.ROLE;
+import com.webdev.cosmo.cosmobackend.service.api.User;
+import com.webdev.cosmo.cosmobackend.service.user.UserService;
+import org.junit.jupiter.api.Test;
+import org.openapitools.model.UserModel;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+
+import java.time.LocalDateTime;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.Mockito.when;
+
+public class UserIT extends BaseTestConfiguration {
+
+    @MockBean
+    private UserService userService;
```

Comment:
In IT we do not want to test only controller layer, please rollback this chages and i will provide simple how to in upcoming days :smile:

Reply:
hmm maybe not rollback, but declare it as USerControllerTests in standard test folder, then if so use mockito instead of @MockBean